Userland/Libraries/LibWeb/HTML/SourceBuffer.cpp

```
@@ -0,0 +1,119 @@
+
+double SourceBuffer::appendWindowEnd() const
+{
+    return m_appendWindowEnd;
+}
+
+void SourceBuffer::setAppendWindowEnd(double end)
+{
+    m_appendWindowEnd = end;
+}
+
+void SourceBuffer::appendBuffer(const BufferSource& buffer)
```

Cppcheck (2.19.0)

[style] The function 'appendBuffer' is never used.

(unusedFunction)